docset/windows/failoverclusters/add-clustervirtualmachinerole.md

```diff
@@ -37,7 +37,11 @@ By creating clustered virtual machines, you can consolidate multiple servers on
 Instead, if that server, or cluster node, fails or requires scheduled maintenance, then another node begins to run the virtual machines instead through a process known as failover.
 The virtual hard disk (VHD) file for the clustered virtual machine must be on the clustered disk used by that virtual machine.
 
-Note: This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
+
+> [!NOTE]This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
+
+> [!NOTE]This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For more information, please go to: https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344
```

Comment:
```suggestion
> [!NOTE] This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For more information, please go to: https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344
```